internal/batches/docker/context.go

```diff
@@ -94,3 +95,20 @@ func fastCommandTimeout() (time.Duration, error) {
 
 	return fastCommandTimeoutData.timeout, fastCommandTimeoutData.err
 }
+
+// executeFastCommand creates a fastCommandContext used to execute docker commands
+// with a timeout for docker commands that are supposed to be fast (e.g docker info).
+func executeFastCommand(ctx context.Context, args ...string) ([]byte, error) {
```

Comment:
can the other commands we run against docker that currently use `withFastCommandContext` also use this helper or would that not work?

Reply:
It would. I wanted to do that in a separate PR incase there are other considerations to be made.
I'll do that in another PR.